packages/cookie-handler/src/style.scss

```diff
@@ -0,0 +1,68 @@
+$ft-cookie-disclaimer-background-color: #cdcdcd !default;
+$ft-cookie-disclaimer-color: #333333 !default;
+$ft-cookie-disclaimer-padding: 20px !default;
+$ft-cookie-disclaimer-transition: 0.3s ease-in !default;
+$ft-cookie-disclaimer-box-max-width: 500px !default;
+$ft-cookie-disclaimer-small-breakpoint: 500px !default;
+$ft-cookie-disclaimer-box-margin: 20px !default;
+
+.ft-cookie-disclaimer {
```

Comment:
Should this be in here or extracted to Craft 3 Blueprint? In my opinion it should be - or at the very least the class name must be updated.